ios/http/FileWebServer.h

```diff
@@ -0,0 +1,35 @@
+//
+//  WebServer.h
+//  PPSSPP
+//
+//  Created by Daniel Gillespie on 7/25/15.
+//
+
+#import <Foundation/Foundation.h>
+
+// Networking (GET IP)
+#include <ifaddrs.h>
+#include <arpa/inet.h>
+
+
+#import "ios/http/Reachability/Reachability.h"
+
+
+// Web Server
+#import "ios/http/GCDWebUploader/GCDWebUploader.h"
+
+
+
+GCDWebUploader *webServer;
```

Comment:
It's incorrect for this to be in the header. Any and all files that import this header will have their own copy instead of using a single shared copy. Did you mean to have `extern GCDWebUploader *webServer;` here, with `GCDWebUploader *webServer;` in the .m file?